library/src/test/java/com/datatorrent/lib/parser/XmlParserTest.java

```diff
@@ -65,6 +73,19 @@ protected void finished(Description description)
   }
 
   @Test
+  public void testOperatorSerialization()
```

Comment:
@shubham-pathak22  this test is only to check if the operator is kryo serializable. If operator's test run a dag then serialization automatically gets validated in unit tests. However, all the tests for this operator directly call process on operator input port, so serialization never got checked. Hence added the test. 
But yes, @chandnisingh had mentioned there could be a utility for this test going forward.